pandera/typing/formats.py

```diff
@@ -49,4 +52,5 @@ class Formats(Enum):
     Literal[Formats.feather],
     Literal[Formats.parquet],
     Literal[Formats.pickle],
+    Literal[Formats.custom],
```

Comment:
Yeah I tried without Literal first, without the literal, the Union below throws an error, even if you remove the Literal from around the Formats.csv for example.

You get this error: TypeError: Union[arg, ...]: each arg must be a type. Got <Formats.csv: 'csv'>.

Checked it with the Literal and using pd.read_csv as the from_format argument and it worked?

Reply:
Another option that is working would be to have this in the Union: `type(Formats.custom)`

Reply:
I guess I'm still confused... can't [this](https://github.com/unionai-oss/pandera/blob/main/pandera/typing/config.py#L53) be updated to `Optional[Union[Format, Callable]]`?

Then there's no need to extend the `Format` Union type, since the `to/from_format` callable will be provided at the `Config` definition.

Reply:
Ah good call thanks, updated!